coralproject/cay
# Service Worker, first version.

## What does this PR do?

It adds a service worker that provides caching of resources. For now assets and other resources will be cached but not API calls.

## How do I test this PR?

- Enter to cay
- disconnect from the internet or use devtools to simulate a really bad network
- refresh
- It should load really fast

@coralproject/frontend
